src/coordination/coordinator_instance.cpp

```diff
@@ -573,6 +575,16 @@ auto CoordinatorInstance::RegisterReplicationInstance(CoordinatorToReplicaConfig
     return RegisterInstanceCoordinatorStatus::FAILED_TO_OPEN_LOCK;
   }
 
+  if (!raft_state_.CoordinatorExists(config_.coordinator_id)) {
+    auto const self_c2c_config = CoordinatorToCoordinatorConfig{
+        .coordinator_id = config_.coordinator_id,
+        .bolt_server = io::network::Endpoint{"0.0.0.0", static_cast<uint16_t>(config_.bolt_port)},
+        .coordinator_server = io::network::Endpoint{"0.0.0.0", static_cast<uint16_t>(config_.coordinator_port)}};
+    if (!raft_state_.AppendAddCoordinatorInstanceLog(self_c2c_config)) {
+      spdlog::error("Failed to append add coordinator instance log for self!");
+    }
+  }
```

Comment:
Why IP 0.0.0.0?

Reply:
this is for adding myself. 0.0.0.0 is better than 127.0.0.1. This is the only reason